phpstan-baseline.neon

```diff
@@ -960,51 +925,44 @@ parameters:
 			path: Tests/Security/Authentication/Authenticator/OAuthAuthenticatorTest.php
 
 		-
-			message: "#^Unreachable statement \\- code above always terminates\\.$#"
+			message: "#^Method FOS\\\\OAuthServerBundle\\\\Util\\\\Random\\:\\:generateToken\\(\\) has no return type(|hint) specified\\.$#"
 			count: 1
-			path: Tests/Security/Authentication/Provider/OAuthProviderTest.php
-
+			path: Util/Random.php
```

Comment:
Apparently there's been an effort to keep these in alphabetical order by file path, so probably a good idea to keep up with that.